js/test/rive.test.ts

```diff
@@ -1124,6 +1125,59 @@ test("Mouse events are removed from canvas when reset", (done) => {
   });
 });
 
+test("Canvas has listener events attached on a Rive file with Rive Listeners", (done) => {
```

Comment:
Oh good call! We can listen for `removeEventListener`. I think I recently discovered you can't query what event listeners are setup on a DOM element directly, so that might work